### PR TITLE
Remove watch, use extra callback to allow CDN to be notified on subse…

### PR DESCRIPTION
**Pull request by cliftonc**
…quent builds

### CODE REVIEW

**Comment by cliftonc** [comment]
As per https://github.com/tes/bosco/issues/167

File: commands/cdn.js [hunk 1]
```diff
@@ -131,114 +129,12 @@ function cmd(bosco, args) {
...
-        reloading[fileKey] = true;
-        reloadFile(fileKey, f);
-      }
-
-      monitor.on('changed', onChange);
-      monitor.on('created', onChange);
-    });
+  function watchCallback(err, service) {
+    if (err) { return bosco.error(err); }
+    bosco.log('Local CDN ready after build for service: ' + service.name.green);
```
Comment (geophree) [inline commands/cdn.js]: Don't we need to re-minify etc every time this is called?

Comment (cliftonc) [inline commands/cdn.js]: This only matters if Bosco if you are running `bosco cdn minify`.  It removes a lot of code if we say that watching the 'old style' projects where there is more than one file in a bundle (e.g. site-assets) doesn't work in minify mode.  If just using `bosco cdn` normally you don't need to do anything here.

**Comment by geophree** [comment]
:+1:

**Pull request merged**